codalab/model/bundle_model.py

```diff
@@ -848,7 +848,10 @@ def transition_bundle_starting(self, bundle, user_id, worker_id):
             # Check if the designated worker is going to be terminated soon
             row = connection.execute(
                 cl_worker.select().where(
-                    and_(cl_worker.c.worker_id == worker_id, cl_worker.c.is_terminating == False)
+                    and_(
+                        cl_worker.c.worker_id == worker_id,
+                        cl_worker.c.is_terminating == False,  # NOQA
```

Comment:
Same with this one. There is no error code and comment explaining why we are ignoring the rule.